### ISSUE TITLE
Axios GET query to GitHub Jobs API returns nothing

### ISSUE THREAD

**Issue by mumorgue404** [mumorgue404/job-filter-app#3]
In src/actions/jobs.js, we are currently loading dummy jobs JSON array from public/jobs.json to avoid making repeated API calls to Github (see lines 19-22 in jobs.js). App functions completely as expected with this setup, but obviously this means we are only ever seeing jobs from the static JSON array we have stored.

However, when commenting this axios call out and uncommenting the one above that is intended to make actual GET request to Github Jobs API, nothing is being returned. 

We need to look into the cause of this and fix it before we can modify the codebase and iterate on its design, as future features depend on this base functionality. I'm not sure what the cause of this is but a good starting point might be looking into if Axios / Redux are being set up and used correctly. 

Looking through the comments here: https://dev.to/myogeshchavan97/build-an-amazing-job-search-app-using-react-42p may also be helpful, to see if anybody else is having the same issue. If it comments to it, A/B comparison with the codebase here https://github.com/myogeshchavan97/github-jobs-react-app should find some difference (you could simply pull latest commit from here). 

If none of this works, it may simply be an issue with the way the Express server is being configured and with how the app is being initialised locally using yarn run start.

**Comment by mumorgue404** [comment]
**Update**

_Reproduction steps_
1. Start Express server by executing "yarn start" from the server/ folder
2. Start React application by executing "yarn start" in another Terminal in the project root folder
3. Attempt to get jobs list by going to localhost:5000/jobs in your browser

_Expected result_
- Scrollable of all jobs is shown with details, graphics, etc.

_Actual result_
- Nothing is shown; blank screen, no blue coloured background
- Looking in browser console we see "the loading of a resource at http://localhost:5000/favicon.ico (“default-src”)." error

**Suspected cause**
From what I understand, there are 2 possible causes of this:
1. The favicon.ico the router is looking isn't where it thinks it is, making the diagnostic message highly misleading (see here: https://stackoverflow.com/questions/64614336/why-do-i-get-the-default-src-none-content-security-policy-error-on-react-pw).  This may be because Express doesn't serve the root folder by default and PUBLIC_URL is misconfigured. If this is the case, the solution may involve manually configuring the absolute path to public/favicon.ico in server/server.js
2. Minor typo in the code resulting in favicon.ico not being able to be located
3. Incomplete routing and error handling for non-existent pages
(see https://stackoverflow.com/questions/64614336/why-do-i-get-the-default-src-none-content-security-policy-error-on-react-pw for options 2 and 3).
Basically it has nothing to do with CSP at all and this error message is very misleading. 

I feel like the routing in server.js is incomplete and a robust solution will involve setting up with custom error handling / config for Express

**Comment by mumorgue404** [comment]
@Alexxpas
